src/main/java/com/commercetools/sync/sdk2/commons/utils/CustomUpdateActionUtils.java

```diff
@@ -329,7 +331,7 @@ List<ResourceUpdateActionT> buildNonNullCustomFieldsUpdateActions(
 
     final String oldCustomTypeId = oldCustomFields.getType().getId();
 
-    // Object is JsonNode
+    // Convert Object to JSONNode
```

Comment:
Where do you convert? Maybe move this comment closer to the place where you do the conversion?

Reply:
[a5de830](https://github.com/commercetools/commercetools-sync-java/pull/987/commits/a5de8303e2ce7084a05ce7343a1dd0baac0528ff)